projects/admin/src/app/record/detail-view/document-detail-view/document-detail-view.component.ts

```diff
@@ -112,21 +112,20 @@ export class DocumentDetailViewComponent implements DetailRecord, OnInit, OnDest
     const keysTypes = Object.keys(this.mappingtypes);
     const rec = record.metadata;
     const route = ['/records', 'documents', 'new'];
-    if (rec.identifiedBy) {
-      const queryParams = [];
-      rec.identifiedBy.forEach((identifier: any) => {
-        if (['bf:Isbn', 'bf:Issn', 'bf:Doi', 'bf:Lccn', 'bf:IssnL', 'bf:Ean'].includes(identifier.type)) {
-          queryParams.push(this._extractAndFormatQueryParams(identifier));
-          if (keysTypes.indexOf(identifier.type) > -1) {
-            const cidentifier = cloneDeep(identifier);
-            cidentifier.type = this.mappingtypes[cidentifier.type];
-            queryParams.push(this._extractAndFormatQueryParams(cidentifier));
-          }
+    let query = null;
+    const queryParams = [];
+    rec.identifiedBy.forEach((identifier: any) => {
+      queryParams.push(this._extractAndFormatQueryParams(identifier));
+      if (keysTypes.indexOf(identifier.type) > -1) {
+        const cidentifier = cloneDeep(identifier);
+        cidentifier.type = this.mappingtypes[cidentifier.type];
+        queryParams.push(this._extractAndFormatQueryParams(cidentifier));
       }
-      });
-      const query = queryParams.join(' OR ');
+    });
+    if (queryParams.length > 0) {
+      query = queryParams.join(' OR ');
       this._recordService.getRecords(
```

Comment:
We could simply add a condition `if (queryParams)` here to fix the problem without searching on all ientifiers. (cf. @benerken)